STANDARDS.md

```diff
@@ -19,54 +69,60 @@ Each dataset shall be versioned.
 
     `"OpenFF <descriptive and uniquely-identifying name> v<version number>"`
 
-- The first submission of a dataset will have a version `"v1.0"`
+- The first submission of a dataset will have a version `"v3.0"`
 
-- A dataset with the suffix `"-beta"` is not to be used for production work.
+* The major version shall indicate the STANDARDS that the dataset conforms to. Datasets which are not intended to conform to any STANDARDS should start with 0, e.g. `"v0.1"`. 
 
-- A minor version change (e.g. `"v1.1"`) means cosmetic or minor additions/problems were addressed
-    - mispelling
-    - addition of a e.g. Wiberg bond orders
+* Datasets with versions starting with `"v1.x"` and `"v2.x"` do not follow any official STANDARDS, and thus should be considered `"v0.x"`.
 
-## Tags indicate status
+- A minor version change (e.g. `"v3.1"`) represents a minor addition and/or fixes problems:
+	- Errors/bugs in the molecule specification
+	- Changes necessary to adhere to the STANDARDS (i.e. changes necessary to placate the NONCOMPLIANT status)
 
-A tag `"complete"` indicates that a dataset is completed as far as OpenFF is concerned.
-This means that any errors remaining are known to be acceptable or impossible to fix.
-It also means that no additional work is being done on the dataset to get it to completion.
+- A patch (e.g. `"v3.1.1"`) represents a cosmetic change, or a change that is based on dynamic information that does not change the underlying data:
+    - Cosmetic changes
+	- Updating the blacklist
+	- Updating the dataset status
 
-A tag `"inflight"` indicates that a dataset is not completed as far as OpenFF is concerned.
-This means that any errors remaining are being actively addressed.
+(It might be useful to make the version bump in the changelog, but not the actual dataset name; this would depend on the ability to changing a dataset metadata of an existing dataset. This would allow existing scripts to not need to update to use a new dataset name when e.g. we update the status to COMPLETE/DONE or NONCOMPLIANT/PAUSED->INCOMPLETE/WORKING).
```

Comment:
Yeah, patch changes probably shouldn't be reflected in a dataset name, since many of these dynamic information bits make more sense as edit-in-place changes to mutable data. I think the best-effort paragraph below gives us enough operational room to handle the kinds of cases we need without being overly descriptive on the implementation path.